test/integration/builtin/tags/if.ts

```diff
@@ -64,6 +64,12 @@ describe('tags/if', function () {
       const html = await liquid.parseAndRender(src, scope)
       return expect(html).to.equal('XY')
     })
+    it('should evaluate promise', async function () {
+      const src = `{%if var == 'var' %}success{%endif%}`
+      const scope = { 'var': Promise.resolve('var') }
+      const html = await liquid.parseAndRender(src, scope)
+      return expect(html).to.equal('success')
+    })
```

Comment:
It would be even better if you also add a case demonstrating `{{ var }}` also works.